src/group/managed_group/mod.rs

```diff
@@ -105,16 +141,58 @@ impl ManagedGroup {
         credential_bundle: &CredentialBundle,
         members: &[usize],
     ) -> Result<Vec<MLSMessage>, ManagedGroupError> {
-        unimplemented!()
+        let mut plaintext_messages: Vec<MLSPlaintext> = members
+            .iter()
+            .map(|member| {
+                self.group.create_remove_proposal(
+                    &self.aad,
+                    &credential_bundle,
+                    LeafIndex::from(*member),
+                )
+            })
+            .collect();
+
+        // Include pending proposals into Commit
+        let mut messages_to_commit = self.pending_proposals.clone();
+        messages_to_commit.extend_from_slice(&plaintext_messages);
+
+        // Create Commit over all proposals
+        let (commit, _, kpb_option) =
+            self.group
+                .create_commit(&self.aad, &credential_bundle, &messages_to_commit, false)?;
+
+        // Add the Commit message to the other pending messages
+        plaintext_messages.append(&mut vec![commit]);
+
+        // If it was a full Commit, we have to save the KeyPackageBundle for later
+        if let Some(kpb) = kpb_option {
+            self.own_kpbs.push(kpb);
+        }
+
+        // Convert MLSPlaintext messages to MLSMessage and encrypt them if required by
+        // the configuration
+        let mls_messages = self.plaintext_to_mls_messages(plaintext_messages);
+
+        Ok(mls_messages)
     }
 
     /// Creates proposals to add members to the group
     pub fn propose_add_members(
```

Comment:
This function doesn't seem to be used and should get a test.

Reply:
Fixed.